fmriprep/cli/parser.py

```diff
@@ -0,0 +1,411 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""Parser."""
+import os
+import sys
+from .. import config
+
+
+def _build_parser():
+    """Build parser object."""
+    from functools import partial
+    from pathlib import Path
+    from argparse import (
+        ArgumentParser,
+        ArgumentDefaultsHelpFormatter,
+    )
+    from packaging.version import Version
+    from .version import check_latest, is_flagged
+    from niworkflows.utils.spaces import Reference, SpatialReferences, OutputReferencesAction
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unused variable 'SpatialReferences'](https://app.codacy.com/manual/poldracklab/fmriprep/pullRequest?prid=5112966)

Reply:
Yes, I have the commit done, was just waiting for Circle to finish so that the build didn't get cancelled.

I have changed Codacy's settings so that only one summary comment will be posted.